### PR TITLE
Pr/590 - new feedback page and about page update

### PR DESCRIPTION
**Pull request by beets**
from #590

### CODE REVIEW

File: server/templates/static/about.html [hunk 1]
```diff
@@ -103,6 +104,23 @@ <h2>Advisory Board</h2>
...
     </ul>
   </section>
+
+  <section>
+    <h2>Origins</h2>
+    <p>
+      Data Commons arose from a set of <a
+        href="https://www.nitrd.gov/nitrdgroups/index.php?title=Open_Knowledge_Network">meetings</a>
+      around this concept, organized by Chaitan Baru of NSF, Andrew Moore
+      (then of CMU) and R.V.Guha, in 2016 and 2017. That eventually lead to
```
Comment (tjann) [inline server/templates/static/about.html:114]: Space before Guha?

File: server/templates/static/feedback.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      the issue.
+    </p>
+    </section>
+
+  <section>
+    <h2>Contributing Data</h2>
+    <p>
+    We are actively looking to partner with Government organizations,
+    Universities and NGOs to add authoritative data. If you are
+    interested, please email us at collaborations at datacommons.org
```
Comment (tjann) [inline server/templates/static/feedback.html:52]: datacommons dot org
?

File: server/templates/static/feedback.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    </section>
+
+  <section>
+    <h2>Other Participation</h2>
+    <p>
+    Data Commons has benefited greatly from our collaborations with
+    different government organizations and academic institutions and are
+    looking to expand the set of collaborative projects. In particular,
+    we are looking for partners creating new tools, building curriculum
+    using Data Commons, for improving documentation and more.
```
Comment (tjann) [inline server/templates/static/feedback.html:63]: `In particular,
    we are looking for partners for creating new tools, building curriculum
    using Data Commons, improving documentation, and more.`
?

**Review by tjann — APPROVED** [review]

File: server/templates/static/about.html (hunk 1 above)
Comment (beets) [reply server/templates/static/about.html:114]: that seems to be how he writes his name, so i didn't want to update it. see guha.com

**Pull request merged**